Calling TestPackagePrivateClassNew.test():
Caught java.lang.reflect.InvocationTargetException
  caused by java.lang.IllegalAccessError
Calling TestPackagePrivateClassNewArray.test():
Caught java.lang.reflect.InvocationTargetException
  caused by java.lang.IllegalAccessError
